module/User/src/Form/ApiPrincipal.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+declare(strict_types=1);
+
+namespace User\Form;
+
+use Laminas\Form\Element\MultiCheckbox;
+use Laminas\Form\Element\Submit;
+use Laminas\Form\Element\Text;
+use Laminas\Form\Form;
+use Laminas\I18n\Translator\TranslatorInterface;
+use Laminas\InputFilter\InputFilterProviderInterface;
+use Laminas\Validator\StringLength;
+use User\Model\ApiPrincipal as ApiPrincipalmodel;
+use User\Model\Enums\ApiPermissions;
+
+use function array_map;
+use function in_array;
+
+class ApiPrincipal extends Form implements InputFilterProviderInterface
+{
+    public function __construct(
+        private readonly TranslatorInterface $translator,
+    ) {
```

Comment:
Because we always use `\Laminas\Mvc\I18n\Translator` we generally do not use the interface (to prevent using `\Laminas\I18n\Translator\Translator`).

Reply:
b48bbb2e9966b14482d8bd2b22ee043892724aac